python/ql/src/Expressions/ExpectedMappingForFormatString.ql

```diff
@@ -13,7 +13,14 @@
 import python
 import semmle.python.strings
 
-from Expr e, ClassObject t
-where exists(BinaryExpr b | b.getOp() instanceof Mod and format_string(b.getLeft()) and e = b.getRight() and
-mapping_format(b.getLeft()) and e.refersTo(_, t, _) and not t.isMapping())
+from Expr e, ClassValue t
```

Comment:
Reformat with four-space indents, and we're good to go!

Reply:
okiedoke. the autoformatter puts in 2 so this will have to be manually done any time this query gets autoformatted fwiw

Reply:
@BekaValentine we have automation for this :muscle: see https://github.com/github/codeql-python-team/issues/33